inst/examples/exm-strat_ests_barplot.R

```diff
@@ -0,0 +1,31 @@
+library(dplyr)
+library(ggplot2)
+
+xs_data <-
+  sees_pop_data_pk_100
+
+curve <-
+  typhoid_curves_nostrat_100 %>%
```

Comment:
```suggestion
  typhoid_curves_nostrat_100 |>
```

Please switch to the base pipe in changed files, following updated guidance from the style guide: https://style.tidyverse.org/pipes.html